### DISCUSSION TITLE
[Feature Request] Is there any way to show completion item detail popup without clicking on the arrow?

### DISCUSSION THREAD

**Discussion by LyqXMU** [microsoft/monaco-editor#4301] [Q&A]
### Context

- [X] This issue is not a bug report. *(please use a different template for reporting a bug)*
- [X] This issue is not a duplicate of an existing issue. *(please use the [search](https://github.com/microsoft/monaco-editor/issues) to find existing issues)*

### Description

When comptition item's detail (or documentation) is too long, users need to click the arrow on the right to open the detail popup. Is there any way to make  the detail popup visible automatically when an item is selected?

![20230824-153955](https://github.com/microsoft/monaco-editor/assets/37861956/3a8267cd-4570-4eb1-abfd-3c62de9e4802)
![20230824-154117](https://github.com/microsoft/monaco-editor/assets/37861956/d89da970-8f98-494e-a582-959eaf2656f2)

### Monaco Editor Playground Link

https://microsoft.github.io/monaco-editor/playground.html?source=v0.40.0#example-extending-language-services-completion-provider-example

**Comment by hediet** [comment]
You can press ctrl+space to toggle the details view. I'm sure there is a command for that.

**Comment by LyqXMU** [comment]
> You can press ctrl+space to toggle the details view. I'm sure there is a command for that.

I'm using mac, but both ctrl+space and cmd+space don't work……

**Reply by rcjsuen** [reply]
Control+Space works for me on a Mac. I am on Chrome.

**Comment by LyqXMU** [comment]
> You can press ctrl+space to toggle the details view. I'm sure there is a command for that.

And I can't find the command, could you please tell me where to find it?

**Comment by SammyWhamy** [comment]
I am looking for this feature too, is there any way?

**Comment by hgrourou** [comment]
someone has solutions?
